gtests/net/mptcp/unsupported/test_client_1.pkt

```diff
@@ -0,0 +1,30 @@
+// connect() function, connection initiated by the kernel
+--tolerance_usecs=100000
+`../../common/defaults.sh`
```

Comment:
@matttbe I usually use the packetdrill directly. The resaon for " ../../" not "../" is that I put the tests in a folder in gtests/net/mptcp/unsupported where "unsupported" is the <dir> where all the tests go. I had to make the changes `../../common/defaults.sh` because the script wouldn't run otherwise.
If this is not okay, kindly let me know and I'll move the tests one folder back.

Reply:
> @matttbe I usually use the packetdrill directly. The resaon for " ../../" not "../" is that I put the tests in a folder in gtests/net/mptcp/unsupported where "unsupported" is the
> where all the tests go. I had to make the changes `../../common/defaults.sh` because the script wouldn't run otherwise.
> If this is not okay, kindly let me know and I'll move the tests one folder back.

oh so you mean your test was in `gtests/net/mptcp/unsupported/<EXTRA_FOLDER>/test_client_1.pkt`? In this case, yes, you need an extra `../` to reach `gtests/net/mptcp/common` dir.
But in this PR, where it is currently located, you don't need the extra `../`

    cd gtests/net/mptcp/unsupported
    ../../packetdrill/packetdrill -v test_client_1.pkt